contracts/PvpArena.sol

```diff
@@ -696,20 +744,36 @@ contract PvpArena is Initializable, AccessControlUpgradeable {
         characterRankingPoints[characterID] = newRankingPoints;
     }
 
+    function _getCharacterShieldRoll(uint256 characterID)
```

Comment:
This is not a "roll" though. Word comes from rolling the dice (meaning randomness is involved)